peppy/project.py

```diff
@@ -259,6 +259,9 @@ def create_samples(self, modify: bool = False):
         """
         self._samples: List[Sample] = self.load_samples()
         if modify:
+            if self.samples is None:
+                _LOGGER.info("No samples found in file.")
+                sys.exit(1)
```

Comment:
I would suggest to create new exception with message no samples found in the project for modification. And don't use sys.exit, because we are using peppy not only for files. We can have peppy as object, and sys.exit can cause issues in downstream projects.

Reply:
I see, ok, I will take it out. 
I made a mistake and put it in if modify condition, it should have been after samples were loaded but before modify. Just wanted to add a more helpful message in case their sample_annotation.csv file had no samples. I'll correct that. Thank you.